data/typelibrary/utils-1.0.0/typelib/splitting/DWORD_TO_WORDS.fct

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Function Name="DWORD_TO_WORDS" Comment="">
+	<Identification Standard="61499-1" Description="Copyright (c) 2024 HR Agrartechnik GmbH    &#10;    &#10;This program and the accompanying materials are made     &#10;available under the terms of the Eclipse Public License 2.0     &#10;which is available at https://www.eclipse.org/legal/epl-2.0/     &#10;     &#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Version="1.0" Author="franz" Date="2024-02-22">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="Normal Execution Request">
+				<With Var="IN"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="Execution Confirmation">
+				<With Var="OUT_00"/>
+				<With Var="OUT_01"/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="DWORD" Comment="DWORD Input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT_00" Type="WORD" Comment="WORD 00"/>
+			<VarDeclaration Name="OUT_01" Type="WORD" Comment="WORD 01"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION DWORD_TO_WORDS
+
+VAR_INPUT
+	IN : DWORD;    // DWORD Input
+END_VAR
+
+VAR_OUTPUT
+	OUT_00 : WORD;    // WORD 00
+	OUT_01 : WORD;    // WORD 01
+END_VAR
+
+(* this Function extracts the 2 WORD from a dword *)
+OUT_00 := DWORD_TO_WORD(SHR(IN AND DWORD_WORD_00, SHIFT_WORD_00));
```

Comment:
In my opinion, you should replace the constants with the values in the shift function, as they are perfectly clear in this context

Reply:
I agree when it is 2 WORDS from a DWORD 

when it is 32 QUARTER from a LWORD, i like the Constants more.

Reply:
In this case, you could also use the .%W<n> access operator to solve this
OUT_00 := IN.%W0;
OUT_00 := IN.%W1;

Although I usually prefer constants, in the case of the SHL/SHR functions, I don't see the benefit to put the number 0-31 into verbose Constants, but if you want to keep them, that's ok for me.
Please just name them all the same then, some start with SHIFT_ and some with SHFT_

Reply:
yes, we will use the Bit Access wherever possible. 

it is easier to read. 

@MoritzO02